### ISSUE TITLE
Consider switching from ovn-nbctl to ovsdbapp

### ISSUE THREAD

**Issue by jsitnicki** [openvswitch/ovn-scale-test#153]
Using `ovn-nbctl` to talk to NBDB has drawbacks:
* we have to scrape the free-from output from `ovn-nbctl`, the output format can change,
* we have to spawn a new process and a new connection for each batch of requests,
* we cannot take advantage of incremental notifications from the OVSDB ("update2").

If parsing the output of `ovn-nbctl` becomes too cumbersome, or generating workload with it turns out to be not efficient enough we should consider switching to ovndbapp, a client library for OVSDB used by OpenStack:

https://github.com/openstack/ovsdbapp

**Comment by l8huang** [comment]
The OVN northbound DB's workload is not that high, and generally we don't need to parse the output of ovn-nbctl verbosely, the exit code is good enough for most of cases. Also ovn-nbctl has a '--wait' option for avoid keep on polling ovsdb, I'm not sure if it's good enough for you use case. 

Definitely, compare to OVSDB client, using ovn-nbctl is more heavy and not flexible, but for scalability test, IMO, it's easy to use and good enough.

**Comment by jsitnicki** [comment]
@l8huang ACK, this is not a priority for us ATM. We might want to do it in the future to be as efficient in stressing the NB DB as OpenStack/OpenShift who use/will use client libraries for OVSDB.

I've opened this issue as a follow up to review discussion for #147, where I suggested that we should avoid parsing free-form output of `ovn-nbctl` and instead ask for machine readable output like CSV or JSON. 

Feel free to close this issue and I will reopen if needed.
